server/views/pages/tasklist.html

```diff
@@ -70,6 +70,7 @@ <h1 class="govuk-heading-m">Prisoner details</h1>
                 <div class="govuk-heading-m">Offending history</div>
                 <div class="govuk-!-margin-top-2">{% if offendingHistoryCompleted %}Completed{% else %}Not yet checked{% endif %}</div>
             </li>
+            {% if not data.isFemale %}
             <li class="app-task-list__item">
```

Comment:
As this is now disabled, going forward db values won't be populated anymore.
Have you searched for every occurrence of the data saved off here, to make sure it is nullsafe if anywhere else is used?

Reply:
The database string isn't there if further charges isn't filled in and isn't called anywhere.